src/engine/Engine.h

```diff
@@ -719,6 +719,13 @@ class Engine : public IEngine, public SignalHandler::Signalable
     */
     bool minimizeCostWithGurobi( const LinearExpression &costFunction );
 
+    /*
+      Get Context reference
+    */
+    Context &getContext() { return _context; }
+
+    bool consistentBounds() const;
```

Comment:
This method doesn't seem to be used anywhere.

Reply:
Both methods are used in the SmtCore, getContext() in the ctor and consistentBounds() on line 287.